src/Map/src/Bridge/Google/assets/src/map_controller.ts

```diff
@@ -121,21 +121,47 @@ export default class extends AbstractMapController<
         });
 
         if (infoWindow) {
-            this.createInfoWindow({ definition: infoWindow, marker });
+            this.createInfoWindow({ definition: infoWindow, element: marker });
         }
 
         return marker;
     }
 
+    protected doCreatePolygon(
+        definition: PolygonDefinition<google.maps.Polygon, google.maps.InfoWindowOptions>
+    ): google.maps.Polygon {
+        const { points, title, infoWindow, rawOptions = {}, extra } = definition;
+
+        const latLngs = points.map((point) => ({ lat: point.lat, lng: point.lng }));
+
+        const polygon = new _google.maps.Polygon({
+            ...rawOptions,
+            paths: latLngs,
```

Comment:
`points` is already a list of `{ lat: number; lng: number }`, so it should be enough no?

```suggestion
        const polygon = new _google.maps.Polygon({
            ...rawOptions,
            paths: points,
```